server/models/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Sequelize } from 'sequelize-typescript';
+import { Chapter } from './chapter';
+
+export const initSequelize = () =>
+  new Sequelize({
+    dialect: 'postgres',
+    username: 'postgres',
+    password: 'password',
+    database: 'chapter',
```

Comment:
i think these are better to pass as env variables to the docker-compose instance and use `process.env.DB_USERNAME` or equivalent here (env vars can then be set for production)

Reply:
Also, same for all the fields